bcgov/moh-prime
# PRIME-2021 Paper Enrolment Status Should Not Be Under Review

[PRIME-2021](https://bcgovmoh.atlassian.net/jira/software/c/projects/PRIME/boards/1?modal=detail&selectedIssue=PRIME-2021)

### How to test:

1. Login as admin
2. Start a paper enrolment by going to `/paper-enrolment/0/demographic`
3. Go through a few pages in the enrolment then leave without completing it
4. Go to admin panel (You should see the paper enrolment with an **Incomplete** status in Orange)
5. Continue the paper enrolment by going to  `/paper-enrolment/<enrolment-id>/demographic`
6. Finish the enrolment
7. Go back to admin panel (this time you should see the paper enrolment status changed to **Complete** and in Blue)

You can also change the status of the paper enrolment to Locked, Declined ...etc. and the status should change accordingly and displaying the correct colours.

Do a regular enrolment stop in the middle and check (Status should be Editable) the go back and complete it (Status should change to Under Review) showing that regular enrolments are unaffected.
